app/assets/stylesheets/components/_contact_panels.scss

```diff
@@ -0,0 +1,90 @@
+.l-contact-panel {
+  @include column(4);
```

Comment:
This needs to default to full width and break to a three column layout at the appropriate breakpoint. Currently the panels display at 1/3 width for all viewport widths.

Reply:
These styles are not needed anymore and got removed